core/src/commonMain/kotlin/com/ustadmobile/core/model/FileToZip.kt

```diff
@@ -0,0 +1,7 @@
+package com.ustadmobile.core.model
+
+
+data class FileToZip(
```

Comment:
This data class should be within the UseCase interface - as per coding style doc.